userguide/plugins.rst

```diff
@@ -223,8 +223,8 @@ command line of the jail's shell instead of from the GUI.
 
 .. _Deleting Plugins:
 
-Deleting Plugins
-----------------
+Deleting
+--------
 
 When you install a plugin, an associated jail is created. If you
```

Comment:
Do not use "you".  In the second sentence, reword to avoid the if/then construct.